### PR TITLE
Miscallleneous stuff

### PR DESCRIPTION
**Pull request by gemelen**
Few changes to cover small gaps.
Most important is `unit` literal, required for `for-expression` even in naive implementation.

### CODE REVIEW

**Comment by gemelen** [comment]
@maxbrunsfeld could you please have a look? (and also to #26)

File: grammar.js [hunk 1]
```diff
@@ -751,6 +755,14 @@ module.exports = grammar({
       $._automatic_semicolon
     ),
 
+    null_literal: $ => token('null'),
```
Comment (maxbrunsfeld) [inline grammar.js:758]: I don't think you need the `token()` call around the string literal.

That function is only needed when you're using other rule-builder functions like `seq`/`repeat`/`choice`, and you want to tell Tree-sitter that the whole rule should be a single token. With a string literal, it's already a single token by default.

**Review by maxbrunsfeld — COMMENTED** [review]
This looks good. I just left one minor suggestion.

**Comment by maxbrunsfeld** [comment]
Thanks!

**Pull request merged**